core-services/prow/02_config/open-cluster-management/go-template-utils/OWNERS

```diff
@@ -0,0 +1,22 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

Comment:
I'm not sure what the effect of leaving these lines in--it's probably safer to remove them for now and let the bot add them in.

Reply:
Sure, no problem.